prefs-compiler/src/main/resources/org/jraf/android/prefs/compiler/prefwrapper.ftl

```diff
@@ -59,7 +60,21 @@ public class ${prefWrapperClassName} extends SharedPreferencesWrapper {
     <#if !disableNullable && pref.defaultValue == "null">
     @Nullable
     </#if><#t>
+    @Deprecated
```

Comment:
That's not very nice! Generating deprecated code. Please remove it, the rest looks good, 'll accept the PR.